src/firebase.js

```diff
@@ -5,26 +5,14 @@ import * as config from './config';
 
 
 let messaging = null;
-let supported = false;
 
 export const notificationApiIsSupported = () =>
 	'Notification' in window &&
 	'serviceWorker' in navigator &&
 	'PushManager' in window
 
-const initializeFirebase = async () => {
-	if (notificationApiIsSupported()) {
-		supported = await isSupported();
-		if (supported) {
-			initializeApp(config.FIREBASE);
-			messaging = getMessaging();
-		}
-	}
-	console.log("Supported", supported);
-};
-
 export async function register() {
-	if (supported && 'serviceWorker' in navigator) {
+	if (isSupported() && 'serviceWorker' in navigator) {
```

Comment:
`isSupported()` returns a `Promise`, so this condition will always be truthy.
```suggestion
	if ((await isSupported()) && 'serviceWorker' in navigator) {
```

(This also applies everywhere else `isSupported` is called)